apps/web/src/pages/studio-onboarding/index.tsx

```diff
@@ -48,7 +43,7 @@ export const StudioOnboarding = () => {
             To start sending your first workflows, you first need to connect Novu to your Bridge Endpoint. This setup
             will create a sample Next.js project and pre-configured the @novu/framework client for you.
           </Text>
-          <SetupTimeline testResponse={{ data, isLoading }} />
+          <SetupTimeline testResponse={{ data: data as BridgeStatus, isLoading }} />
```

Comment:
I am a bit surprised we need this assertion here.

Reply:
I was to but typescript refused to let it pass without this :(